packages/workflow/src/ErrorReporterProxy.ts

```diff
@@ -0,0 +1,33 @@
+export interface IReportingOptions {
+	tags: Record<string, string>;
+	extra: Record<string, unknown>;
+}
+
+interface IErrorReporter {
```

Comment:
Can we please move away from the `I-` style? I'll post references supporting this later in the channel.

Reply:
I agree. This was just me trying to imitate the code in other files in the same folder. sounds like this should be a lining rule.